frontend/src/features/mapSideBar/containers/MapSideBarContainer.test.tsx

```diff
@@ -92,7 +92,7 @@ describe('Parcel Detail MapSideBarContainer', () => {
       });
     });
 
-    it('parcel sidebar snapshot loads by id', async () => {
+    xit('parcel sidebar snapshot loads by id', async () => {
```

Comment:
Did you want to ignore this?

Reply:
yes, this test is for a component that isn't displayed anymore. it will be removed in an upcoming pr.